### PR TITLE
 TFX MLMD Client Library proposal

### PR DESCRIPTION
**Pull request by casassg**
Initial draft for MLMD Client library proposal.

Addresses #12

### CODE REVIEW

**Comment by casassg** [comment]
@googlebot I signed it!

**Comment by rcrowe-google** [comment]
`tfx/addons/metadata` seems a bit generic, could we do something like `tfx/addons/metadata-client` ?

**Comment by casassg** [comment]
> `tfx/addons/metadata` seems a bit generic, could we do something like `tfx/addons/metadata-client` ?

I like it. Ammended proposal

File: proposals/20210507-mlmd_client_lib.md
Comment (pselden) [inline proposals/20210507-mlmd_client_lib.md:61]: Could you add paul.selden@openx.com (or I can open a PR on top of your PR if you'd prefer that)

File: proposals/20210507-mlmd_client_lib.md
Comment (codesue) [inline proposals/20210507-mlmd_client_lib.md:59]: sue@twitter.com

**Comment by rcrowe-google** [comment]
Approved by Google team, awaiting approval by core team.

**Comment by casassg** [comment]
@rcrowe-google Could we approve via email in core team? Similar to apache votes? https://www.apache.org/foundation/voting

**Comment by rcrowe-google** [comment]
@hanneshapke

**Comment by rcrowe-google** [comment]
**Pending review by core team:**

@casassg @hanneshapke @michalbrys @pselden @rclough @junbopark @sngahane @marcromeyn

Folks that I don't have Github usernames for (please send):
daikeshi@gmail.com larry.price@openx.com nle@twitter.com olgai@spotify.com

**Please review by 5pm PT Saturday June 12, 2021**

**Review by rcrowe-google — APPROVED** [review]
Approved by both Google and Core team

**Pull request merged**
